src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+function centerSpider(first, second) {
+  const positionX
+    = (parseFloat(getComputedStyle(first).width)
```

Comment:
You can use clientWidth and clientHeight properties. It's generally more efficient and will simplify the code structure